src/Dialog.js

```diff
@@ -37,7 +38,7 @@ export default class Dialog extends Component {
         {this.props.name ?
           <h1 className={css.title}>{this.props.name}</h1>
         : null}
-        <div className={css.body}>
+        <div className={classNames(css.body, getScrollingDialogClassname())}>
```

Comment:
It does not change -- I exported this function so the CreateIssueDialog could access the class name. Doesn't need to be a function, just a const.